packages/grenier/grenier.0.12/opam

```diff
@@ -7,7 +7,7 @@ license: "ISC"
 dev-repo: "git+https://github.com/let-def/grenier.git"
 doc: "https://let-def.github.io/grenier/doc"
 build: [
-  ["dune" "subst"] {pinned}
+  ["dune" "subst"] {dev}
```

Comment:
Well spotted. @kit-ty-kate do you know if we can extend the lint action to check for this?

Reply:
the issue with adding a lint check for this is that half the packages that are autogenerated and will have this issue until they upgrade to dune >= 2.7. It's good to do for some packages but not for all (also given that most packages inside the opam-repository right now uses `{pinned}`)